### PR TITLE
[9.0][website_payment_v10] Backport website_payment from v10.

### PR DESCRIPTION
**Pull request by yajo**
Rationale on this backport can be found on #204.

This is **not compatible** with `website_portal` from v9.

Please somebody mark as WIP until these are merged:
- [x] #204 

@Tecnativa

### CODE REVIEW

**Comment by rafaelbn** [comment]
Please rebase

File: website_payment_v10/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+warnings.
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Uninstall ``website_portal``, if it was installed, and don't install it
+   again as long as this module is installed.
+
```
Comment (rafaelbn) [inline website_payment_v10/README.rst]: Uninstall `website_portal` ? Are you sure ;-)

**Comment by rafaelbn** [comment]
Please check travis, it's failing.

Functionally :+1:  Thanks!

**Comment by carlosdauden** [comment]
:+1:

**Pull request merged**